### DISCUSSION TITLE
Where should I place an .npmrc file?

### DISCUSSION THREAD

**Discussion by 8thHalfHour** [coreybutler/nvm-windows#568] [General] [answered]
Where should I place a `.npmrc` file for a specific Windows profile? The user's home directory? Also, will it be applied to any version of node I switch to, or is there a way to limit which versions the `.npmrc` applies itself to?

**Comment by coreybutler** [comment, marked as answer]
You can use any configuration [supported by npm](https://docs.npmjs.com/files/npmrc#files). NVM4W doesn't touch npm (other than installing npm), so it won't touch any npm config files. If you set a `.npmrc` file in the user's home directory, it would apply to any version of node/npm the user switches to.
